firefox-ios/Client/Frontend/TabContentsScripts/LoginsHelper.swift

```diff
@@ -136,6 +136,13 @@ class LoginsHelper: TabContentScript, FeatureFlaggable {
               let type = res["type"] as? String
         else { return }
 
+        // NOTE: This is added only to be able t oswitch between implementations inside the JS.
+        // Once we rollout the update for all users, this will be removed.
```

Comment:
Do we need a follow up ticket here?

Reply:
Yes. Added in https://github.com/mozilla-mobile/firefox-ios/pull/26164/commits/f280a0ce30cf1105f03a7126913f4363db996122 😄